common/djangoapps/student/models.py

```diff
@@ -1442,6 +1442,28 @@ def enrollments_for_user(cls, user):
         return cls.objects.filter(user=user, is_active=1)
 
     @classmethod
+    def preloaded_enrollments_for_user(cls, user):
+        """
```

Comment:
[10:47 AM] David Ormsbee: how about "enrollments_with_overviews_for_user" as a method name?
@robrap: How about `enrollments_with_overviews_for_user`? I realize it doesn't capture that it may not always fully pre-load the overviews, but I feel like that's a detail that's better expressed in docstring than the method name -- and it will preload CourseOverviews the vast, vast majority of the time.